requirements.txt

```diff
@@ -7,7 +7,7 @@ httpx>=0.27.0
 importlib-resources
 kubernetes>=28.1.0
 mmh3>=4.0.1
-numpy>=1.22.5
```

Comment:
let's keep both restrictrions,

`numpy>=1.22.5, <2.0.0

Reply:
you're absolutely right, fixed, thanks!